platform-sdk/swirlds-state-impl/src/main/java/com/swirlds/state/merkle/StateUtils.java

```diff
@@ -477,29 +479,55 @@ public static <K> Bytes getVirtualMapKeyForKv(
     }
 
     /**
-     * Creates Protocol Buffer encoded byte array for a VirtualMapKey field.
+     * Creates an instance of {@link VirtualMapValue} which is stored in a {@link com.swirlds.virtualmap.VirtualMap}.
```

Comment:
"... which is stored in a state". VirtualMap can store any bytes. This method is only used by readable/writable states